servicetalk-concurrent-internal/src/main/java/io/servicetalk/concurrent/internal/DelayedCancellable.java

```diff
@@ -52,4 +52,11 @@ public void cancel() {
             oldCancellable.cancel();
         }
     }
+
+    /**
+     * Ignores any subsequent calls to {@link #cancel()}, preventing propagating the cancellation further up the stream.
+     */
```

Comment:
```suggestion
    /**
     * Ignores any subsequent calls to {@link #cancel()}, preventing propagating the cancellation further up the stream.
     * Note: if {@link DelayedCancellable#delayedCancellable(Cancellable)} is called after this method the provided
     * {@link Cancellable} will be immediately cancelled to prevent leakage of resources.
     */
```